src/mercury_engine_data_structures/samus_returns_data.json

```diff
@@ -0,0 +1,798 @@
+        "fields": {
+            "fMaxTimeAlive": "float"
+        }
+    },
+    "CTunableDroppable": {
+        "kind": "struct",
+        "parent": "CTunable",
+        "fields": {
+            "fSpawnSpeedFactor": "float",
+            "fMaxSpawnVelocity": "float"
+        }
+    },
+    "CDroppableMissileComponent": {
+        "kind": "struct",
+        "parent": "CDroppableComponent",
+        "fields": {
+            "sItemMax": "string",
+            "sItemCurrent": "string"
+        }
+    },
+    "CDroppableSuperMissileComponent": {
+        "kind": "struct",
+        "parent": "CDroppableComponent",
+        "fields": {
+            "sItemMax": "string",
+            "sItemCurrent": "string"
+        }
+    },
+    "CTutorialComponent": {
+        "kind":"struct",
+        "parent": "CComponent",
+        "fields": {}
+    },
+    "CTutorialMissileAimingComponent": {
+        "kind":"struct",
+        "parent": "CTutorialComponent",
+        "fields": {
+            "fMinTimeToShowBlockMessage": "float",
+            "sChargeFX": "string"
+        }
+    },
+    "CTutorialMissileComponent": {
+        "kind":"struct",
+        "parent": "CTutorialComponent",
+        "fields": {}
+    },
+    "CMovementComponent": {
+        "kind":"struct",
+        "parent": "CComponent",
+        "fields": {
+            "bIsFlying": "bool"
+        }
+    },
+    "CADNMovementComponent": {
+        "kind":"struct",
+        "parent": "CMovementComponent",
+        "fields": {}
+    },
+    "CTunableADNComponent": {
+        "kind":"struct",
+        "parent": "CTunable",
+        "fields": {}
+    },
+    "CADNScannerUserComponent": {
+        "kind":"struct",
+        "parent": "CComponent",
+        "fields": {}
+    },
+    "CTunableADNScannerUserComponent": {
+        "kind":"struct",
+        "parent": "CTunable",
+        "fields": {
+            "fFollowADNCameraMinZ": "float",
+            "fFollowADNCameraMaxZ": "float",
+            "fFollowADNCameraMinZDistance": "float",
+            "fFollowADNCameraMaxZDistance": "float",
+            "fFollowADNCameraNoSceneOffsetDistance": "float",
+            "fFollowADNCameraExitInterp": "float",
+            "fFollowADNCameraExitInterpTime": "float",
+            "fFollowADNCameraExitInterpInSceneTime": "float",
+            "sScanMatFX": "string",
+            "fFirstTimeStateFadeStepTime": "float",
+            "fFirstTimeStateFadedIdleTime": "float"
+        }
+    },
+    "CBombFanComponent": {
+        "kind":"struct",
+        "parent": "CComponent",
+        "fields": {
+           "sWindFX": "string" 
+        }
+    },
+    "CHazarousPoolComponent": {
+        "kind": "struct",
+        "parent": "CComponent",
+        "fields": {
+            "fSpeed": "float",
+            "fMaxSpeed": "float",
+            "fFillSpeed": "float",
+            "fAcceleration": "float",
+            "fTotalTime": "float",
+            "sDrainSound": "string",
+            "_sFanfareSound": "string"
```

Comment:
Seems wrong

Reply:
Looks like I copied it by accident. I'll double check it later